structure-proposal.md

```diff
@@ -0,0 +1,104 @@
+# Modular Base-Structure for the Simulation Study
+
+## Goal
+
+The primary goal of this project is to develop a new (adversarial) collaboration approach for simulation studies, aiming to bolster their generalizability by delegating critical, subjectivity-prone decision-making processes when conducting a simulation based study to neutral or adversarial parties. To establish a foundational framework within which such an adversarial collaborative approach can be developed and showcased the following aims to identify the structure of all decision-making elements that can be subjected to such collaboration in simulation studies. 
```

Comment:
I think we had slightly different goals. I did not yet focus on identifying the structure of elements that can be subjected to collaboration, but rather the general structure of simulation studies on its own